commander/migrate_test.go

```diff
@@ -48,6 +59,16 @@ func (s *MigrateTestSuite) SetupTest() {
 	s.cmd.storage = s.storage.Storage
 
 	setStateLeaves(s.T(), s.storage.Storage)
+
+	s.pendingBatches = []dto.PendingBatch{
+		makePendingBatch(1, models.TransferArray{testutils.MakeTransfer(0, 1, 0, 100)}),
+		makePendingBatch(2, models.TransferArray{testutils.MakeTransfer(0, 1, 1, 100)}),
+	}
+
+	s.failedTxs = models.MakeTransferArray(
```

Comment:
We can create `failedTxs` directly in `TestMigrateCommanderData_SyncsFailedTxs` as it's used only there.